core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -524,6 +529,7 @@ service FileSystemMasterClientService {
    */
   rpc ListStatus(ListStatusPRequest) returns (stream ListStatusPResponse);
 
+  rpc LoadMetadata(LoadMetadataPRequest) returns (LoadMetadataPResponse);
```

Comment:
Can you comment this rpc and also leave the space between this definition and the next RPC definition